src/scss/styles.scss

```diff
@@ -0,0 +1,54 @@
+@import "variables.scss";
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+@import "./components/header";
+@import "./components/buttons";
+@import "./components/icons";
+@import "./components/forms";
+
+main{
+    min-height: 100vh;
+}
+
+h1, h2, h3, h4, h5{
+    color: $gray-800;
+}
+
+.img-thumbnail{
+    max-width: 70px;
+    max-height: 70px;
+    border: 0;
+    padding: 0;
+    cursor: pointer;
+}
+
+a{
+    text-decoration: none;
+    color: $black;
+
+    &:hover{
+        text-decoration: none;
```

Comment:
Accessibility concern. Links should have clear state indicators such as a change in colour, underlining, etc.